hikari/files.py

```diff
@@ -473,6 +473,38 @@ async def read(
         async with self.stream(executor=executor) as reader:
             return await reader.read()
 
+    async def save(
+        self,
+        *,
+        path: typing.Optional[typing.Union[pathlib.Path, str]] = None,
```

Comment:
This should use Pathish.

Reply:
You can later just cast the path to `pathlib.Path` and use that directly.

`path.exists`, `path.expanduser`, `path.open` and so on.

Reply:
Yeah I was thinking of doing that. I tend to handle it that way in my own projects.